src/ubuntu/18.04/amd64/Dockerfile

```diff
@@ -56,6 +56,7 @@ RUN apt-get update \
     && apt-get install -y \
         autoconf \
         automake \
+        curl \
```

Comment:
I think we should keep building the curl from sources. There was a reason for that, I think it was to get libcurl built with the right APIs.

Reply:
in the very old days, we used curl directly in HttpClient and it did matter what SSL backend was used. I don't think this is the case any more. 
That change is 4 years old touching libcurl-openssl1.0-dev as well.

Reply:
Ok, then I'm fine with it.